manifests/s/Synology/SurveillanceStationClient/2.2.1-2564/Synology.SurveillanceStationClient.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Synology.SurveillanceStationClient
+PackageVersion: 2.2.1-2564
+PackageLocale: en-US
+Publisher: Synology
+PublisherUrl: https://www.synology.com
+PublisherSupportUrl: https://www.synology.com/company/contact_us
+PrivacyUrl: https://www.synology.com/privacy
+PackageName: Surveillance Station Client
+License: Copyright (c) Synology Inc.
+ShortDescription: Protect your business or home with a reliable and intuitive surveillance solution with powerful and intelligent video monitoring, management, and analysis tools.
+Moniker: synologysurveillancestationclient

```

```suggestion
Moniker: synology-surveillance-station
```

Would this work better as a more readable friendly name?

---

AFAIK the Moniker `synologysurveillancestationclient` would serve no purpose here since it already matches the PackageName (case and space insensitively) and I believe `winget install synologysurveillancestationclient` _should_ work regardless. If you add hyphens it could enable a command like `winget install synology-surveillancestation-client` to work additionally & could be a more readable way as well

---

makes sense! sounds good to me